PIDs:
  X:
    kp: 25.0
    kd: 14.0
    ki: 0.2
    minOutput: -10.0
    maxOutput: 10.0
    integratorMin: -0.1
    integratorMax: 0.1
  Y:
    kp: 25.0
    kd: 14.0
    ki: 0.2
    minOutput: -10.0
    maxOutput: 10.0
    integratorMin: -0.1
    integratorMax: 0.1
  Z:
    kp: -30000.0
    kd: -15000.0
    ki: -3000.0
    minOutput: 10000.0
    maxOutput: 60000.0
    integratorMin: -1000.0
    integratorMax: 1000.0
    kp1: -41000.0
    kd1: -10000.0
    ki1: -7000.0
    minOutput1: -10000.0
    maxOutput1: 10000.0
    integratorMin1: -1000.0
    integratorMax1: 1000.0
  Yaw:
    kp: 500.0
    kd: 100.0
    ki: 0.1
    minOutput: -200.0
    maxOutput: 200.0
    integratorMin: 0.0
    integratorMax: 0.0
